python/src/PosteriorDistribution_doc.i.in

```diff
@@ -1,23 +1,33 @@
 %feature("docstring") OT::PosteriorDistribution
 "Distribution conditioned by observations.
 
-Available constructors:
-    PosteriorDistribution(*conditional, observations*)
-
 Parameters
 ----------
 conditional : :class:`~openturns.ConditionalDistribution`
-    The conditional distribution.
+    The conditional distribution of density :math:`f_{\vect{X}}(\vect{x}) = \int f_{\vect{X}|\vect{\Theta}=\vect{y}}(\vect{x}|\vect{y}) f_{\vect{Y}}(\vect{y})\di{\vect{y}}`.
```

Comment:
```suggestion
    The distribution of density :math:`f_{\vect{X}}(\vect{x}) = \int f_{\vect{X}|\vect{\Theta}=\vect{y}}(\vect{x}|\vect{y}) f_{\vect{Y}}(\vect{y})\di{\vect{y}}`.
```
I know the class is named that way but the name is badly chosen, so let's try to keep the confusion to a minimum.